### PR TITLE
add dockerfile and necessary changes

### LINKED ISSUE #182: Deploy app

### PR DESCRIPTION
**Pull request by NLmeng**
closes #182

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="A Personalized Trip Planning App"
```
Comment (L0Lmaker) [inline client/public/index.html:10]:
```suggestion
      content="Create the Trip of Your Dreams"
```

File: server/index.js [hunk 2]
```diff
@@ -42,9 +45,9 @@ if (config.server.env === 'DEV') {
 
 app.use(errorHandleMiddleware)
 
-app.get('/', (req, res) => {
-  res.send('Hello, world!')
-})
+app.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/build', 'index.html'));
+});
```
Comment (L0Lmaker) [inline server/index.js:50]: does it make a difference if we change from '/' to '*'?

**Review by L0Lmaker — COMMENTED** [review]
small comments

**Review by AndyLiang1 — APPROVED** [review]

File: server/index.js (hunk 2 above)
Comment (NLmeng) [reply server/index.js:50]: '/' is the root URL, '*' is all other path that was not defined in our backend

**Pull request merged**